### DISCUSSION TITLE
Paragraph Numbering

### DISCUSSION THREAD

**Discussion by MaPeras** [knowledgefutures/pubpub#1926] [Support]
Dear folks,

As I have mentioned in previous posts, German legal academia loves its footnotes, and we like to cite as specific as possible. One example - while citing legal cases - is its focus on paragraph numbering. This would be a nice way to cite PubPub-Articles without relying on a PDF version in the background. For an example, see [this judgement](https://www.bundesverfassungsgericht.de/SharedDocs/Entscheidungen/EN/2020/05/rs20200505_2bvr085915en.html#abs1) of the German constitutional court. The main body of the judgement starting "With their constitutional complaints..." has the section number 1 on the right side. 

Is there a way to integrate automated numbering? Linking to specific parts of paragraphs already works pretty nice! We could add  paragraph numbers to each section manually, but inserting/updating the text down the road is a real pain. 

Paradise would be a possibility to cite a sentence from a paragraph, generating the correct citation while marking the text as it currently works with linking ("copy a permanlink"). 

Looking forward to hints and opinions.

Cheers
Max

**Comment by gabestein** [comment]
Hello! A few questions here as we start to scope this out:

- What about non-paragraph but non-figure/media elements, e.g. lists. Would those need to be numbered as well?
- Where would numbers be displayed, ideally? Left gutter? Right gutter?
- Would they always be displayed, or would there be an option to see them on hover or something like that?
- In terms of linking, does the current paragraph link functionality suffice?

**Comment by MaPeras** [comment]
Hi Gabe,

> What about non-paragraph but non-figure/media elements, e.g. lists. Would those need to be numbered as well?

I would say yes, if you mean the list as a whole. A paragraph number for each bullet point would be a bit too much.

> Where would numbers be displayed, ideally? Left gutter? Right gutter?

Maybe the left gutter as comments are on the right? However, you are the UI-design experts

> Would they always be displayed, or would there be an option to see them on hover or something like that?

I guess being displayed always sounds good. I imagine myself looking for a specific paragraph that was cited elsewhere - it might be easier to have some visual aid.

> In terms of linking, does the current paragraph link functionality suffice?

Not sure, if I understand your question, but I think the current possibility of linking is quite wonderful. Do you mean, does it suffice instead of paragraph numbering? If you only publish in digital formats, it might work, but many disciplines still prefer paper and there you need some other kind of reference (like paragraph numbering).

Thank you for your engagement with the topic.

Cheers
Max

**Reply by MaPeras** [reply]
Please give me a hint, if you need any further information. I am happy to discuss it.

**Reply by augustfly** [reply]
Jumping into this discussion. I think for us the problem is that our community also uses section numbering widely. And our articles on ingest can automatically reflect (or be made to refer to) Sections by number. But there are no numbers anywhere in the final document reflecting this (see screenshot). This is very confusing for a reader, who goes looking for Section 9, but has no idea which is the 9th section. 

We normally put Section numbers in the left gutter. In our other journals we section number to the subsubsection (6.3.1), but even high level sections would be valuable. 

![Screenshot 2024-01-24 at 8 50 16 AM](https://github.com/pubpub/pubpub/assets/234283/ed623697-753c-4768-a162-fce1cbd38787)
